syncer/syncer.go

```diff
@@ -1360,6 +1386,14 @@ func (s *Syncer) syncDML(
 		case sqlJob, ok := <-jobChan:
 			metrics.QueueSizeGauge.WithLabelValues(s.cfg.Name, queueBucket, s.cfg.SourceID).Set(float64(len(jobChan)))
 			if !ok {
+				tctx.L().Info("close job chan", zap.Any("rest job", jobs))
+				affect, err = executeSQLs()
```

Comment:
L1390-L1396 seems appear many times, can we put them in one function

Reply:
or maybe move the logic for`executing the remaining tasks in the queue ti downstream` to`s.closeJobChans`

Reply:
> L1390-L1396 seems appear many times, can we put them in one function

Can you help me think of a function name?

Reply:
I just noticed something a little different，why does the L1388 need to continue

Reply:
> I just noticed something a little different，why does the L1388 need to continue

Because the `jobs` may not clean completely.

Reply:
in `fatalF` jobs will clear by `clearF`, i think't we don't need into another round of for loop, because jobChan already closed

Reply:
My bad, it shall not be `continue`.